main.c

```diff
@@ -112,19 +113,40 @@ void print_rolls(int *dice_nums) {
     if(!print_separate) printf("\n");
 }
 
+/* roll_from_interactive() - parse stdin, one roll by line, don't exit on errors
+ *
+ * Parameters: None
+ * Returns: None
+ */
+void roll_from_interactive(){
+     int dice_nums[DICE_ARRAY_SIZE] = {};
+     static char *line = (char *)NULL;
+
+     line = readline("");
```

Comment:
If this is to be an interactive mode, perhaps we should use a standard prompt (like "> "), to make it clear?